appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.29.2",
+    "version": "0.29.3",
```

Comment:
I already bumped the version in https://github.com/Microsoft/vscode-azuretools/pull/396 and I can just include it in that